instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -1028,16 +1031,24 @@ def client_response_hook(send_span, scope, message):
             )
 
 
-def get_distribution_with_fastapi(*args, **kwargs):
-    dist = args[0]
-    if dist == "fastapi~=0.58":
-        # Value does not matter. Only whether an exception is thrown
-        return None
-    raise DistributionNotFound()
+def mock_version_with_fastapi(*args, **kwargs):
+    req_name = args[0]
+    if req_name == "fastapi":
+        # TODO: Value now matters
```

Comment:
What's the TODO here? It's something we should fix later?

Reply:
This is from Jeremy, I think it means it should be updated if the one in pyproject changes.